src/index.ts

```diff
@@ -7,6 +7,33 @@ function init() {
   */
 }
 
+/*
+ * @param cards - Array of cards
+ * @returns maxHeight of all cards that are displayed on the page as Integer
+ * this iterates over array of cards and reads the height of each card to get the maximum height of all cards
+ */
+function getMaxHeight(cards) {
+  let maxHeight = 0;
+  cards.forEach((card) => {
+    const cardHeight = card.offsetHeight;
```

Comment:
Its not wrong in this case. But if we have cards with border, we would use .clientHeight instead of .offsetHeight.

https://medium.com/@jbbpatel94/difference-between-offsetheight-clientheight-and-scrollheight-cfea5c196937